chart/testdata/frobnitz/templates/wordpress.jinja

```diff
@@ -0,0 +1,72 @@
+#helm:generate dm_template
```

Comment:
This is test data. The idea was to have one particular isolated example that we can run tests against. Ideally, it only changes when we need to change it for test integrity. So we can't just pull in live data from somewhere else.